modules/etcd-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="etcd-overview_{context}"]
+= Overview of etcd
+
+etcd is a consistent, distributed key-value store that holds small amounts of data that can fit entirely in memory. Although etcd is a core component of many projects, it is the primary data store for Kubernetes, which is the standard system for container orchestration.
+
+[id="etcd-benefits_{context}"]
+== Benefits of using etcd
+
+By using etcd, you can enjoy several benefits:
+
+* Safely and automatically update containers
+* Coordinate work that is scheduled to hosts
+* Set up overlay networking for containers
+* Maintain consistent uptime for your cloud-native applications, and keep them working even if individual servers fail
+* Store and replicate all cluster states for Kubernetes
+* Distribute configuration data to provide redundancy and resiliency for the configuration of nodes
+
+[id="etcd-architecture_{context}"]
+== How etcd works
+
+To ensure a reliable approach to cluster configuration and management, the etcd Operator manages etcd within the Operator Framework. The Operator simplifies the use of etcd on a Kubernetes container platform like {product-title}. With the etcd Operator, you can create or delete etcd members, resize clusters, perform backups, and upgrade etcd.
```

Comment:
technically, we use the predecessor (?) of the Operator Framework, so I would leave out the mention of it.

On the features of the CEO, I would mention that it takes care of mundane maintenance tasks for the administrator - think of defragmentation, certificate renewals and the other things you've mentioned